morphocluster/frontend/src/views/Project.vue

```diff
@@ -1,11 +1,21 @@
 <template>
     <div id="project">
-        <nav class="navbar navbar-expand-lg navbar-light bg-dark text-light">
-            <router-link class="navbar-brand text-light" to="/">MorphoCluster</router-link>
-            <ul class="navbar-nav mr-auto">
-                <li class="nav-item active text-light">Projects</li>
-            </ul>
-            <dark-mode-control />
+        <nav class="navbar navbar-expand-lg navbar navbar-dark bg-dark">
+            <a class="navbar-brand" href="/p">MorphoCluster</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+                aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="/p" >Projects</a>
```

Comment:
Never use `<a href="..."/>` links in Vue. Use `<router-link class="nav-link" :to=.../>`.